packages/office-ui-fabric-react/src/components/ScreenReaderAlert/ScreenReaderAlert.Props.ts

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+export enum ReadingMode {
+  /**
+   * Do not read the content at all.
+   */
+  DoNotRead = 0,
+
+  /**
+   * The content in this component will read out after other reading tasks of screen reader.
+   * E.g. first read the information of currently focus control, then this content.
+   * NOTE: ReadingMode.ReadAfterOtherContent doesn't support ChromeVOX v53.
+   */
+  ReadAfterOtherContent = 1,
+
+  /**
+   * Read immediately after this component is triggered rendering and updated.
+   * Most screen readers will announce the word 'alert' before or after the whole sentence.
+   */
+  ReadImmediately = 2
+}
+
+export interface IScreenReaderAlertProps extends React.Props<HTMLElement> {
+  /**
+   * Use readingMode to specify whether to read the alert, or read before/after normal screen reader content.
+   *
+   * @default ReadingMode.ReadAfterOtherContent
+   */
+  readingMode?: ReadingMode;
+
+  /**
+   * Use indicator to specifically tell screen reader this text should be read even when text is not changed.
+   * Especially in case of re-reading the same text or the text that is possibly the same with the old one.
+   *
+   * Change the indicator to read the text no matter the text is changed or not.
+   * Keep indicator the same value or simply not use it to read ONLY when the text is changed.
+   */
+  indicator?: number;
```

Comment:
We basically have three props, 
- text
- readingMode
- indicator

If anyone tries to read a text which is same as the previous one, they need to trigger the re-rendering of this component. Changing props can achieve that but both **text** and **readingMode** are not supposed to be changed in this case. So `indicator` provides a way for component consumers to **re-read** the text.
In theory, as long as you change the value of indicator, the re-rendering and re-read will be triggered. In most use cases, we simply plus 1 to this number to trigger the re-reading. It seems non-obvious pattern but easy to use